main/main.c

```diff
@@ -855,16 +856,18 @@ PHPAPI ZEND_COLD void php_log_err_with_severity(char *log_message, int syslog_ty
 			char *tmp;
 			size_t len;
 			zend_string *error_time_str;
+			size_t error_date_format_len;
```

Comment:
Unused var?

Reply:
Thank you, yes forgot to remove after refactoring this away.